packages/flutter_tools/lib/src/ios/simulators.dart

```diff
@@ -637,6 +639,8 @@ class _IOSSimulatorLogReader extends DeviceLogReader {
     _systemProcess.stdout.transform(UTF8.decoder).transform(const LineSplitter()).listen(_onSystemLine);
     _systemProcess.stderr.transform(UTF8.decoder).transform(const LineSplitter()).listen(_onSystemLine);
 
+    // Callback hookups' Futures are uninteresting.
+    // ignore: unawaited_futures
     _deviceProcess.exitCode.whenComplete(() {
```

Comment:
(and that mentions that nothing here throws -- the reason we don't want to await this isn't that the future is uninteresting, that's a tautology. it's that it's uninteresting because it won't throw.)